Monika After Story/game/script-affection.rpy

```diff
@@ -2060,21 +2099,32 @@ label monika_affection_nickname:
                             m 2dfc "..."
                             m 2lfc "That really hurt."
                             m "A lot more than what you can imagine."
-                            m 2efc "Forget about this idea."
-                            m "It seems it was a mistake."
-                            m 1efc "Let's talk about something else."
+                            if mas_getEV('mas_apology_bad_nickname').shown_count == 2:
+                                m 2efc "Forget about this idea."
+                                m "It seems it was a mistake."
+                                m 1efc "Let's talk about something else."
                             show monika 1efc
                             pause 5.0
                         else:
                             m 4efd "[player]! That's not nice at all!"
                             m 2efc "Why would you say such things?"
                             m 2rfw "If you didn't want to do this, you should've just said so!"
                             m 2dftdc "..."
-                            m 2lftsc "I don't like this idea anymore."
                             m 2ektsc "...You didn't have to be so mean."
                             m 2dftdc "That really hurt, [player]."
-                            m 2efc "Please don't do that again."
+
+                            if  mas_getEV('mas_apology_bad_nickname').shown_count == 2:
+                                m 2efc "Forget about this idea."
+                                m "It seems it was a mistake."
+                                m 1efc "Let's talk about something else."
```

Comment:
You have some repeated dialogue, can we wrap this into another label so we don't have to maintain 2 sections of the same code.